.travis.yml

```diff
@@ -16,7 +16,7 @@ before_install:
  - sudo add-apt-repository -y ppa:terry.guo/gcc-arm-embedded
  - sudo apt-get update
  - sudo apt-get install libsdl1.2-dev gcc-arm-none-eabi
- - curl -Ls http://s3.voneicken.com/xtensa-lx106-elf.tgx | tar Jxf -
+ - curl -Ls http://s3.voneicken.com/xtensa-lx106-elf-20160330.tgx | tar Jxf -
```

Comment:
it's the compiler from a newer version of esp-open-sdk, corresponding to the 2016-03-30 commit of open-sdk.